src/function.py

```diff
@@ -68,6 +74,14 @@
 BACKOFF_MULTIPLIER = 2
 # Max length in bytes of the payload
 MAX_PAYLOAD_SIZE = 1000 * 1024
+# Individual request timeout in seconds (non-configurable)
+INDIVIDUAL_REQUEST_TIMEOUT_DURATION = 3
```

Comment:
Hi @MattWhelan , I'm submitting a PR to the docs size to mention about the `DEBUG_LOGGING_ENABLED` environment variable, as it is not mentioned there. Do you think, though, that it is necessary to document `INDIVIDUAL_REQUEST_TIMEOUT_DURATION` or `SESSION_MAX_PROCESSING_TIME` [here](https://docs.newrelic.com/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/aws-lambda-sending-cloudwatch-logs/#config-retries)? IMHO, I think we don't want to over-complicate the documentation with these details, but I'd like to know your opinion on this. Thank you!

Reply:
I agree; these can stay internal, undocumented params.

Reply:
Thank you!